api/services/CronService.js

```diff
@@ -44,6 +45,91 @@ module.exports = {
 
     });
 
-  }
+  },
 
+  splittedAggregatedDownloadstats :function(days,callback){
+    console.log('Started splitted aggregated download count');
+    ElasticSearchService.lastMonthDownloadsBulk(days,callback);
+   },
+
+  processDownloads:function(response,directDownloads,indirectDownloads,total,callback){
```

Comment:
Looks good, to improve it, I would isolate the function in another service, like `DownloadStatsService` or something like that.

I would also move the undefined check to the function before, so the `processDownloads` have only one responsibility: computing the stats, checking the error is the responsibility of the function that receive the response